### DISCUSSION TITLE
[@types/ioredis] sendCommand is incorrect

### DISCUSSION THREAD

**Discussion by pafik13** [DefinitelyTyped/DefinitelyTyped#59283] [Issues with a @types package]
`sendCommand` has one input parameter, minimum (instance of `Command`). 

* https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ioredis/index.d.ts#L62
* https://github.com/luin/ioredis/blob/master/lib/commander.ts#L134
* https://github.com/luin/ioredis/blob/master/lib/command.ts#L71
* https://github.com/luin/ioredis/blob/master/lib/redis/index.ts#L730

[a comment by an automated account is not shown]

**Comment by plantain-00** [comment]
You can create a PR to fix this.
